lib.php

```diff
@@ -158,7 +145,7 @@ public function supports_ajax() {
      */
     public function extend_course_navigation($navigation, navigation_node $node) {
         global $PAGE;
-        // If section is specified in course/view.php, make sure it is expanded in navigation.
+        // if section is specified in course/view.php, make sure it is expanded in navigation
```

Comment:
To revert. Comments must begin by a capital letter and must end with a punctuation character.